stablehlo/reference/Ops.cpp

```diff
@@ -1034,14 +1065,14 @@ Tensor evalMapOp(ArrayRef<Tensor> inputs, Region &computation, Scope &scope,
                  ShapedType resultType) {
   Tensor result(resultType);
   for (auto it = result.index_begin(); it != result.index_end(); ++it) {
-    SmallVector<Tensor> args;
+    SmallVector<StablehloValue> args;
     for (size_t i = 0; i < inputs.size(); ++i) {
       auto tensor =
           Tensor(computation.getArgument(i).getType().cast<ShapedType>());
       tensor.set({}, inputs[i].get(*it));
-      args.push_back(tensor);
+      args.push_back(StablehloValue(tensor));
     }
-    result.set(*it, eval(computation, args, &scope)[0].get({}));
+    result.set(*it, getTensors(eval(computation, args, &scope))[0].get({}));
```

Comment:
I wonder if we can get rid of getTensors altogether to minimize the API surface. The proposed change to introduce Scope::findTensors addresses most of the uses, and the rest can be reduced to using getTensor down the line, right?

Reply:
Yes, I've removed it in the following commit.